src/main/java/org/templegalaxia/DemuxArtNetDatagram.java

```diff
@@ -0,0 +1,46 @@
+package org.templegalaxia;
+
+import heronarts.lx.color.LXColor;
+import heronarts.lx.model.LXFixture;
+import heronarts.lx.output.ArtNetDatagram;
+
+public class DemuxArtNetDatagram extends ArtNetDatagram {
+  private static final int ARTNET_HEADER_LENGTH = 18;
+  private final int[] pointIndices;
+
+  public static DemuxArtNetDatagram fromFixture(LXFixture fixture, int universe) {
+    return new DemuxArtNetDatagram(LXFixture.Utils.getIndices(fixture), universe);
+  }
+
+  private DemuxArtNetDatagram(int[] pointIndices, int universe) {
+    super(pointIndices, pointIndices.length, universe);
+    this.pointIndices = pointIndices;
+  }
+
+  public byte luminance(int rgb) {
```

Comment:
One question @mcslee , there isn't much documentation on the color model, what are the output ranges of:
`LXColor.`
 - red/green/blue
 - h/s/b
?

Reply:
The color functions all internally use integers in standard 32-bit ARGB format, one byte each for A, R, G, B. Which is what Processing also uses.

The red() / green() / blue() functions return a byte 0-255.

h() is 0-360, s() and b() are 0-100.

I'll make a note to document those all better.

Source reference:
https://github.com/heronarts/LX/blob/master/src/heronarts/lx/color/LXColor.java#L57